### DISCUSSION TITLE
How does euphony know that acoustic-data-communication has started?

### DISCUSSION THREAD

**Discussion by happy-refactoring** [euphony-io/euphony#132] [Q&A] [answered]
In the case of voice recognition,
Commands are recognized after walking up the AI assistant with `Hey Siri`, `Hey Google`, `Hi bixby`, etc.

In the case of euphony, I know that this role is known by looking at a specific frequency.

What exactly is the specific frequency in hz?

**Comment by sunyeongan** [comment]
17619

**Comment by nodobi** [comment]
17814

**Comment by jiwon0829** [comment]
17948

**Comment by phj0407** [comment]
17980

**Reply by phj0407** [reply]
17988

**Comment by zion830** [comment]
18827

**Comment by hyunjung-choi** [comment]
17890

**Comment by jiwon0829** [comment]
17900

**Comment by serim53** [comment]
17920

**Comment by jiwon0829** [comment]
17930

**Comment by Kimhaedam** [comment]
> In the case of voice recognition, Commands are recognized after walking up the AI assistant with `Hey Siri`, `Hey Google`, `Hi bixby`, etc.
> 
> In the case of euphony, I know that this role is known by looking at a specific frequency.
> 
> What exactly is the specific frequency in hz?

17591

**Comment by DahyeonWoo** [comment]
17915

**Comment by serim53** [comment]
17916

**Comment by kldaji** [comment]
17924

**Comment by serim53** [comment, marked as answer]
17914

**Comment by jiwon0829** [comment]
17914

**Comment by yoouyoou** [comment]
17928
